custom-connectors/Datelist/apiDefinition.swagger.json

```diff
@@ -0,0 +1,304 @@
+                  "title": "Calendar",
+                  "format": "int32",
+                  "x-ms-dynamic-values": {
+                    "operationId": "ListCalendars",
+                    "value-path": "id",
+                    "value-title": "name"
+                  }
+                },
+                "url": {
+                  "type": "string",
+                  "description": "url",
+                  "x-ms-notification-url": true,
+                  "x-ms-visibility": "internal",
+                  "title": "URL"
+                }
+              },
+              "required": [
+                "url",
+                "calendar_id"
+              ]
+            }
+          }
+        ]
+      }
+    },
+    "/calendars": {
+      "get": {
+        "responses": {
+          "200": {
+            "description": "default",
+            "schema": {
+              "type": "array",
+              "items": {
+                "type": "object",
+                "properties": {
+                  "id": {
+                    "type": "integer",
+                    "format": "int32",
+                    "description": "id"
+                  },
+                  "name": {
+                    "type": "string",
+                    "description": "name"
+                  }
+                }
+              }
+            },
+            "headers": {
+              "Content-Type": {
+                "description": "Content-Type",
+                "type": "string"
+              },
+              "Accept": {
+                "description": "Accept",
+                "type": "string"
+              }
+            }
+          }
+        },
+        "operationId": "ListCalendars",
+        "x-ms-visibility": "important",
+        "summary": "List Calendars",
+        "description": "List your bookings calendars",
+        "parameters": [
+          {
+            "name": "Accept",
+            "in": "header",
+            "required": true,
+            "type": "string",
+            "default": "application/json",
+            "x-ms-visibility": "internal"
+          },
+          {
+            "name": "Content-type",
+            "in": "header",
+            "required": true,
+            "type": "string",
+            "default": "application/json",
+            "x-ms-visibility": "internal"
+          }
+        ]
+      }
+    }
+  },
+  "definitions": {},
+  "parameters": {},
+  "responses": {},
+  "securityDefinitions": {
+    "Cl\u00e9 API": {
+      "type": "apiKey",
+      "in": "header",
+      "name": "Authorization"
+    }
+  },
+  "x-ms-connector-metadata": [
+    {
+      "propertyName": "Website",
+      "propertyValue": "https://datelist.io"
+    },
+    {
+      "propertyName": "Privacy Policy",
+      "propertyValue": "https://datelist.io/privacy"
+    },
+    {
+      "propertyName": "Categories",
+      "propertyValue": "Communication;Productivity"
+    }
+  ],
+  "security": [
+    {
+      "Cl\u00e9 API": []
```

Comment:
@aclarembeau Unicodes are not allowed in any connector's artifact, Can you please remove them all.

Reply:
@aclarembeau Any update ?